java/impl/gcp/src/main/java/co/worklytics/psoxy/CloudFunctionRequest.java

```diff
@@ -39,14 +44,33 @@ public Optional<String> getQuery() {
         return request.getQuery();
     }
 
+    /**
+     * @return view of Headers with lower-case names
+     *
+     * this is kinda defensive; while AWS seems to pass headers in lower-case, GCP does not. (or
+     * at least docs, and prior practice, suggest they do not). But really we probably want to
+     * presume case-insensitivity, giving clients leeway in how they send headers.
+     */
+    private Map<String, List<String>> getCaseInsensitiveHeaders() {
```

Comment:
No worth having this duplicated method in a utils class or something?

Reply:
it's not exactly duplicated, bc for gcp its `Map<String, List<String>>`; for aws it's `Map<String, String>`